.github/workflows/acceptance.yml

```diff
@@ -81,6 +80,7 @@ jobs:
 
     env:
       VISUAL_SNAPSHOT_ENABLE: 1
+      MIGRATIONS_TEST_MIGRATE: 1 # Not sure if this is needed...
```

Comment:
This isn't needed for acceptance, only backend testing where we want to explicitly run django migrations.